test/verify/check-metrics

```diff
@@ -731,5 +731,81 @@ class TestMultiCPU(MachineCase):
         b.wait(lambda: int(b.text(".pf-c-tooltip .pf-l-flex:nth-child(2) div:nth-child(2)")[:-1]) > 20)
 
 
+@skipImage("no PCP support", "fedora-coreos")
+@skipImage("FIXME: no redis on test image", "fedora-33", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-2004", "rhel-8-5", "rhel-9-0", "centos-8-stream")
+@skipDistroPackage()
+class TestGrafanaClient(MachineCase):
+
+    provision = {
+        "0": {"address": "10.111.112.1/20", "dns": "10.111.112.1"},
+        # forward Grafana port, so that a developer can connect to it with local browser
+        "services": {"image": "services", "forward": {"3000": 3000}}
+    }
+
+    def testBasic(self):
+        m = self.machine
+        mg = self.machines['services']
+
+        # start Grafana
+        mg.execute("/root/run-grafana")
+        m.execute("until curl http://10.111.112.100:3000; do sleep 1; done")
+        # enable PCP plugin; like on Cog (Configuration) menu → Plugins → Performance Co-Pilot → Enable
+        mg.execute("curl --silent --show-error -u admin:foobar -d '' 'http://127.0.0.1:3000/api/plugins/performancecopilot-pcp-app/settings?enabled=true'")
+
+        # enable redis+pmproxy on supervised machine (should become an UI element)
+        if m.image.startswith("debian") or m.image.startswith("ubuntu"):
+            redis = "redis-server"
+        else:
+            redis = "redis"
+        m.execute(f"systemctl enable --now {redis} pmproxy")
```

Comment:
Let's just start it, not enable -- that works on Debian as well through `Alias=redis.service`. It just can't be enabled through an alias. We don't need this to reboot.

Reply:
Done.

Reply:
Nack, a non-running service does not respect Alias= either, this is strictly an `enable` thing. So going back to the original solution.